requirements.txt

```diff
@@ -1,119 +1,3 @@
-# Required for Home Assistant core
-requests>=2.0
```

Comment:
Would it be possible that we keep `requirements.txt` around but just for the core dependencies. Have `setup.py` read the dependencies in from this file for the core.

Move all platform dependencies to `requirements_all.txt`. That way I can still have Docker install them all.